backend/cypress/e2e/search-tests.cy.js

```diff
@@ -0,0 +1,156 @@
+import searchPage from '../pageObjects/searchPage.js';
+
+let search;
+let yearsToCheck;
+let ueiOrein;
+let aln;
+let name;
+let accDate;
+let state;
+let cogOrover;
+let findings;
+let directFunding;
+let majorProgram;
+
+beforeEach(() => {
+  cy.visit('/dissemination/search/');
+
+  cy.fixture('searchPageData/auditYears.json').then((data) => {
+    yearsToCheck = data.yearsToCheck;
+  })
+
+  cy.fixture('searchPageData/uei-ein.json').then((data) => {
+    ueiOrein = data.ueiOrein;
+  })
+
+  cy.fixture('searchPageData/aln.json').then((data) => {
+    aln = data.aln;
+  })
+
+  cy.fixture('searchPageData/name.json').then((data) => {
+    name = data.name;
+  })
+
+  cy.fixture('searchPageData/accDate.json').then((data) => {
+    accDate = data.accDate;
+  })
+
+  cy.fixture('searchPageData/state.json').then((data) => {
+    state = data.state;
+  })
+
+  cy.fixture('searchPageData/cog-over.json').then((data) => {
+    cogOrover = data.cogOrover;
+  })
+
+  cy.fixture('searchPageData/auditFindings.json').then((data) => {
+    findings = data.findings;
+  })
+
+  cy.fixture('searchPageData/directFunding.json').then((data) => {
+    directFunding = data.directFunding;
+  })
+
+  cy.fixture('searchPageData/majorProgram.json').then((data) => {
+    majorProgram = data.majorProgram;
+  })
+
+  search = new searchPage();
+});
+
+//basic search test
+describe('Test Basic Search Fields', () => {
+
+  it('checks Audit Years', () => {
+    //audit years
+    search.checkAuditYearCheckbox('2023');
+    search.uncheckAuditYearCheckbox('2023');
+    search.checkAllYearsCheckbox('all_years');
+
+    //ueiORein
+    search.testUEIorEin(ueiOrein);
+
+    //name
+    search.testName(name);
+
+    //facAcceptanceDate
+    const [startDate, endDate] = accDate;
+    search.testFACacceptanceDate(startDate, endDate);
+
+    //state
+    search.testState(state);
+
+    //submit button
+    search.testSearchSubmitButton();
+
+    //search results
+    search.testSearchTable();
+
+    //summary report
+    search.testSummaryReport();
+
+  });
+
+
+});
+
+//advanced search test
```

Comment:
I think either remove this commented out stuff, or put a comment explaining why it's commented out.